src/dagmc/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+message("")
```

Comment:
Optional: Rather than having a blank message, I would state here that the `dagmc` CMake is starting.  Or remove the message entirely.